samples/nrf9160/modem_shell/src/curl/tool/Makefile.am

```diff
@@ -0,0 +1,155 @@
+#***************************************************************************
```

Comment:
Same story as wit the other Makefile - `samples/nrf9160/modem_shell/src/curl/tool/Makefile` is autogenerated and is very large, please consider removing it.